docs/content/preview/deploy/multi-dc/async-replication/async-transactional-tables.md

```diff
@@ -14,7 +14,7 @@ tags:
 type: docs
 ---
 
```

Comment:
could add a tip here about using automatic replication to avoid having to make manual ddl changes